### DISCUSSION TITLE
Storage policies are not shown

### DISCUSSION THREAD

**Discussion by maxcodefaster** [supabase/supabase#2663] [Questions] [answered]
Hi when i create a storage policies, it is not shown in the storage>policies view.
When i query `select * from pg_policies` i can see the policie though.

How to fix this? 

Thank you.

**Comment by silentworks** [comment, marked as answer]
This is a bug, let me ping someone on the Supabase team and see if it can get resolved soon. 

Hey @inian can you have a look at this bug please?

**Reply by inian** [reply]
We have fixed this now and should be deployed tonight or by tomorrow. Thanks for reporting and bringing this up @maxcodefaster and @silentworks!
